docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md

```diff
@@ -38,14 +38,14 @@ The Intel Enclave Common API library is necessary for creating, initializing, an
 It does not supporting quoting, and consequentially, attestation which is based on quoting. The lack
 of quoting capability is a limitation of SGX1 machines which don't have FLC support.
 
-Firstly we download the Intel SGX DCAP self-extracting executable from [here](http://registrationcenter-download.intel.com/akdlm/irc_nas/16474/Intel%20SGX%20DCAP%20for%20Windows%20v1.5.100.2.exe). Run the executable to unzip files to a specified location.
-The following summary will assume that the contents were extracted to `C:\Intel SGX DCAP for Windows v1.5.100.2`:
+Firstly we download the Intel SGX DCAP self-extracting executable from [here](http://registrationcenter-download.intel.com/akdlm/irc_nas/16605/Intel%20SGX%20DCAP%20for%20Windows%20v1.6.100.2.exe). Run the executable to unzip files to a specified location.
```

Comment:
Please also fix the PSW 2.7 download link on lines 11-13; GitHub won't let me peg a suggestion comment against it directly.